test/unit/context/context_mm_black.h

```diff
@@ -70,17 +74,19 @@ class ContextBlack : public CxxTest::TestSuite {
 
     unsigned factor = 3;
     N = 16384 / factor;
-
     // Push, then allocate, then pop all at once
-    for(unsigned p = 0; p < 5; ++ p) {
+    for (unsigned p = 0; p < 5; ++p)
+    {
       d_cmm->push();
-      for(unsigned i = 1; i < N; ++i) {
-        unsigned len = i * factor;
+      for (unsigned i = 1; i < N; ++i)
+      {
+        len = i * factor;
```

Comment:
This is a unit test, so I don't really care. However, I think that in general, it's better not to reuse a variable for two different things.

Reply:
I could have moved the initialization to the head of the `for` loop, it's really in two separate blocks, and honestly, I don't share your sentiment. But for better readability I didn't move it to the for loop header. I actually don't really want to change this.